tests/unit/dataSource/properties/tests.html

```diff
@@ -1020,10 +1020,12 @@
 	 		
 	 		dsFilter.settings.filtering.filterExprUrlKey = null;
 			var prevMonthDate = new Date(new Date(ticks).setMonth(-1));
-			var prevMonth =prevMonthDate.getMonth() + 1;
-	 		var currMonth = date.getMonth() % 12 + 1;
-			var nextMonth = currMonth % 12  + 1;
-			var nYear = date.getMonth() + 1 === 12 ? date.getFullYear() + 1 : date.getFullYear();
+			var nextMonthDate = new Date(new Date(ticks).setMonth(1));
```

Comment:
setMonth(1) will always set the month to February. Try with this code:
new Date(new Date(ticks).setMonth(new Date(ticks).getMonth() + 1));